docs/gettingstarted.rst

```diff
@@ -49,7 +49,12 @@ Quickstart
       Pull requests welcome!
 
 
-At that point, you should have a functional Socorro development environment.
+At that point, you should have a basic functional Socorro development
+environment.
+
+See :ref:`webapp-chapter` for additional setup and running the webapp.
+
+See :ref:`processor-chapter` for additional setup and running the processor.
```

Comment:
If someone is reading Getting Started, they do some setup and then this connects them to additional bits they'll need to do for the processor and/or the webapp depending on what they want to do now.